code/modules/admin/admin_verbs.dm

```diff
@@ -782,12 +782,12 @@ var/global/list/admin_verbs_mod = list(
 		M.skin_tone =  -M.skin_tone + 35
 
 	// hair
-	var/decl/sprite_accessory/new_hstyle = input(usr, "Select a hair style", "Grooming") as null|anything in decls_repository.get_decls_of_subtype(/decl/sprite_accessory/hair)
+	var/decl/sprite_accessory/new_hstyle = input(usr, "Select a hair style", "Grooming") as null|anything in decls_repository.get_decl_paths_of_subtype(/decl/sprite_accessory/hair)
 	if(new_hstyle)
 		M.h_style = new_hstyle.type
 
 	// facial hair
-	var/decl/sprite_accessory/new_fstyle = input(usr, "Select a facial hair style", "Grooming")  as null|anything in decls_repository.get_decls_of_subtype(/decl/sprite_accessory/facial_hair)
+	var/decl/sprite_accessory/new_fstyle = input(usr, "Select a facial hair style", "Grooming")  as null|anything in decls_repository.get_decl_paths_of_subtype(/decl/sprite_accessory/facial_hair)
 	if(new_fstyle)
 		M.f_style = new_fstyle.type
```

Comment:
See above. Changed here to be a path but two lines down it still does `new_fstyle.type` which is invalid.
```suggestion
		M.f_style = new_fstyle
```